app-backend/tile/src/main/scala/image/GlobalSummary.scala

```diff
@@ -13,12 +12,19 @@ import geotrellis.vector.Extent
 import com.typesafe.scalalogging.LazyLogging
 import cats.data._
 import cats.implicits._
-
 import java.util.UUID
+
+import geotrellis.spark.io.postgres.PostgresAttributeStore
+
 import scala.concurrent._
 import scala.util._
 
 object GlobalSummary extends LazyLogging {
+  val system = AkkaSystem.system
```

Comment:
It doesn't need to be implicit here, trying to verify if it does need to be in the other spots